ReSwift/CoreTypes/Store.swift

```diff
@@ -134,9 +134,15 @@ open class Store<State: StateType>: StoreType {
     }
 
     open func unsubscribe(_ subscriber: AnyStoreSubscriber) {
+        #if swift(<5.0)
```

Comment:
Until https://github.com/apple/swift/commit/e66095b10a8caa8ab568efc36223835120a1f7fc , the compiler directives did not support the `<` operator. This change is only available in Swift 5.0, so while this works in 5.0, it fails in anything below, as seen by the Travis CI jobs.

In order for this to work you'll need to switch it to `#if swift(>=5.0)`, and swap the contents of the if/else.

Reply:
Thank you for telling me.
I'll fix it :)